### PR TITLE
Change HttpRequestHelper.getValueFromParamMap to getRequestParamValue in *Action classes #5572

### LINKED ISSUE #5572: Change HttpRequestHelper.getValueFromParamMap to getRequestParamValue in *Action classes
Keeping the abstraction barrier intact, one less import (and therefore less coupling involved) for those `*Action` classes, and shorter code for line length limit purposes.

### PR DESCRIPTION
**Pull request by Neurrone**
Fixes #5572

### CODE REVIEW

**Comment by Neurrone** [comment]
Ready for review.

**Comment by wkurniawan07** [comment]
@Neurrone there should be more to be removed than just these instances. If they are hiding behind static methods (something like `private static FeedbackQuestionAttributes extractFeedbackQuestionData(Map<String, String[]> requestParameters, ...)`), you can remove the `static` and the `requestParameters` variable, but make sure they do not have any undesired effect.

**Comment by Neurrone** [comment]
Changes made.

**Comment by wkurniawan07** [comment]
@Neurrone can you examine if there are still more instances in `FeedbackSubmissionEditSaveAction.java`? I go by the assumption that you can refactor all the way until you can remove the import `HttpRequestHelper`, but in some cases it may not be possible.

**Comment by Neurrone** [comment]
Changes made.

File: src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java [hunk 1]
```diff
@@ -73,17 +72,16 @@ protected ActionResult execute() throws EntityDoesNotExistException {
...
             if (totalResponsesForQuestion == null) {
                 continue; // question has been skipped (not displayed).
             }
             
             List<FeedbackResponseAttributes> responsesForQuestion = new ArrayList<FeedbackResponseAttributes>();
-            String questionId = HttpRequestHelper.getValueFromParamMap(
-                    requestParameters,
-                    Const.ParamsNames.FEEDBACK_QUESTION_ID + "-" + questionIndx);
+            String questionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_ID + "-"
+                    + questionIndx);
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java]: Line break after `(` is better? i.e. the entire parameter in one line

File: src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java [hunk 2]
```diff
@@ -274,35 +272,27 @@ private FeedbackResponseAttributes extractFeedbackResponseData(
...
-                Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + questionIndx + "-" + responseIndx);
+        response.recipient = getRequestParamValue(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-"
+                + questionIndx + "-" + responseIndx);
         Assumption.assertNotNull("Null feedback recipientEmail", response.recipient);
         
-        String feedbackQuestionType = HttpRequestHelper.getValueFromParamMap(
-                requestParameters,
-                Const.ParamsNames.FEEDBACK_QUESTION_TYPE + "-" + questionIndx);
+        String feedbackQuestionType = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_TYPE + "-"
+                + questionIndx);
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java]: Here, you can line break after `=` 

As a general rule, try to line break at a higher level of the syntax tree.

**Comment by Neurrone** [comment]
@damithc changes made.

File: src/main/java/teammates/ui/controller/InstructorFeedbackQuestionVisibilityMessageAction.java [hunk 3]
```diff
@@ -68,18 +56,14 @@ private static FeedbackQuestionAttributes extractFeedbackQuestionData(
...
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
-
-        String questionType = HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                                     Const.ParamsNames.FEEDBACK_QUESTION_TYPE);
+        newQuestion.showResponsesTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO));
+        newQuestion.showGiverNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO));
+        newQuestion.showRecipientNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/InstructorFeedbackQuestionVisibilityMessageAction.java]: Break at first `(` even better (i.e. higher level)?

**Comment by Neurrone** [comment]
@damithc changes made. What's different about this case compared to the previous instances?

File: src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java [hunk 4]
```diff
@@ -172,67 +165,53 @@ private static FeedbackQuestionAttributes extractFeedbackQuestionData(Map<String
...
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO));
-        newQuestion.showRecipientNameTo = getParticipantListFromParams(
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
+        newQuestion.showResponsesTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO));
+        newQuestion.showGiverNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO));
+        newQuestion.showRecipientNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java]: Sorry, missed this one. :-p

The line break can be at a higher level, just like the other case.

**Pull request merged**